pending_changelog/monitorupdatingpersister.txt

```diff
@@ -0,0 +1,6 @@
+## Backwards Compatibility
```

Comment:
I wrote a top-level comment on the design for up/downgrade

Reply:
(I tried resolving here because this comment is hidden by default when I load the page... feel like I'm fighting GitHub a lot here... anyway, unresolved it)

I'd be happy to set the sentinel value to anything. Matt had suggested two `0xFF` bytes, but we could make it four, or add a descriptive ASCII string as well (at one point, I made it `"LDK-MonitorUpdatingPersister"`).